tests/test_bar/path/xrandr

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+cat << EOF
+Screen 0: minimum 8 x 8, current 1920 x 1200, maximum 16384 x 16384
```

Comment:
(Though I actually *do* have VGA-0 in xrandr output, just it is not connected, so I had to do copy-paste.)